src/foam/nanos/dig/DigWebAgent.java

```diff
@@ -295,27 +295,16 @@ public void execute(X x) {
             foam.lib.json.Outputter outputterJson = new foam.lib.json.Outputter(OutputterMode.NETWORK);
             List a = sink.getArray();
             String dataToString = "";
+            outputterJson.setOutputJsonj_(true);
 
             //resp.setContentType("application/json");
-            for ( int i = 0; i < a.size(); i++ ) {
+            for ( int i = 0 ; i < a.size() ; i++ )
```

Comment:
Does this compile? looks like the for loop is missing the opening {
And is the space before ; part of our style guide?

Reply:
Yes, It is compiled successfully, I just followed this our guide line below.

https://github.com/foam-framework/foam/wiki/FOAM2-Style-Guidelines
----------------------------------------------------------

One-statement if, while, and for statements that fit in less than 80 characters, do not need braces.
if ( ! found ) return false;

for ( var i = 0 ; i < a.length ; i++ ) a[i] = '';